### PR TITLE
fix the package-lock tracking of pp-client 2.94.1

### PR DESCRIPTION
**Pull request by siosonel**
This fixes the package-lock that incorrectly tracked a local dev version of the proteinpaint-client, instead of the published version this morning.

This update also adds the local PP dev server host to the connect-src directive, for SJ dev environments that use local PP server instance.

## Checklist

- [ ] Added proper unit tests
- [ ] Left proper TODO messages for any remaining tasks
- [ ] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

**Review by craigrbarnes — APPROVED** [review]

File: packages/portal-proto/next.config.js [hunk 1]
```diff
@@ -4,6 +4,18 @@
...
+const connectSrc = [
+  "https://portal.gdc.cancer.gov",
+  "https://browser-intake-datadoghq.com",
+  "https://api.gdc.cancer.gov",
+  "https://www.google-analytics.com",
+];
+const PROTEINPAINT_HOST = PROTEINPAINT_API.split("://")[1]?.split("/")[0] || "";
+// in SJ dev environment, this would point to a local PP server instance
+if (!connectSrc.includes(`https://${PROTEINPAINT_HOST}`))
+  connectSrc.push(`https://${PROTEINPAINT_HOST}`);
```
Comment (philtom-ctds) [inline packages/portal-proto/next.config.js:18]: @siosonel I realize this PR already got merged.  Could you submit another PR that limits the PP parts of this code to execute only when `NODE_ENV === 'development'`?  That will prevent unintentionally affecting production.
